structs.go

```diff
@@ -20,8 +20,12 @@ type Struct struct {
 	raw     interface{}
 	value   reflect.Value
 	TagName string
+	TagOptsFieldOmitter tagOptsFieldOmitter
+	IncludeEmptyConversions bool
 }
 
+type tagOptsFieldOmitter func(tags []string) bool
```

Comment:
Is this type here intentionally kept private (ie lowercased)? Seems like since the `TagOptsFieldOmitter` filed on `Struct` is publicly accessible, the type should be as well

Reply:
Alternately, you could not make this a type and just put the function type in the `Struct` above

```
type Struct struct {
    ...
    TagOptsFieldOmitter func(tags []string) bool
    ...
}
```

Reply:
That's a good point. I'll make it public.